src/marks/rule.js

```diff
@@ -103,9 +103,10 @@ export class RuleY extends Mark {
     {width, height, marginTop, marginRight, marginLeft, marginBottom}
   ) {
     const index = filter(I, Y, X1, X2);
+    const {dx, dy} = this;
     return create("svg:g")
         .call(applyIndirectStyles, this)
-        .call(applyTransform, null, Y && y, 0, 0.5)
+        .call(applyTransform, null, Y && y, dx, offset + dy)
```

Comment:
Maybe in cases like this it would be preferable to initialize {dx = 0, dy = offset}, rather than adding offset to dy?

Reply:
No, it's a separate concern. If you want a shape to be 1px to the left of another, they should both be moved by .5px if that helps make them sharper.